### PR TITLE
Making block.shapes easier to read and more specific

### PR DESCRIPTION
**Pull request by TigerbyteDev**
Since Shape is already implemented like that in prismarine-world it makes sense to have it like that here too

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@ interface Effect {
     duration: number;
 }
 
+
+export type Shape = [number, number, number, number, number];
```
Comment (rom1504) [inline index.d.ts:14]: 6 not 5

Comment (TigerbyteDev) [reply index.d.ts]: Thank you!

**Comment by TigerbyteDev** [comment]
@rom1504 hey, in case you didn't see it. I updated it the day after yesterday :)

File: index.d.ts [hunk 2]
```diff
@@ -68,7 +71,7 @@ declare class Block {
      * Each bounding box is an array of the form [xmin, ymin, zmin, xmax, ymax, zmax].
      * Depends on the type and state of the block.
      */
-    shapes: Array<Array<number>>
+    shapes: Shape[][]
```
Comment (rom1504) [inline index.d.ts:74]: You put the array in Shape so it's only one array now

Shape[]

Comment (TigerbyteDev) [reply index.d.ts:74]: oof, I should really stop doing stuff late night. Thank you!

**Pull request merged**